tests/adapters/vip-search/VipSearchPhraseMatchingTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Elasticsearch Extensions Tests: VIP_Enterprise_Search_Adapter_TestAPI class.
+ *
+ * @package Elasticsearch_Extensions
+ * @subpackage Tests
+ */
+use Elasticsearch_Extensions\Factory;
+
+it( 'tests that phrase matching matches phrase exactly.', function () {
+	// Adapter is loaded in the VIP Unit Test Case.
+	// Enable phrase matching.
+	elasticsearch_extensions()->load_adapter( Factory::vip_enterprise_search_adapter() );
+	elasticsearch_extensions()->enable_phrase_matching();
+
+	// Run a search for the phrase "exact match" and get the global wp_query.
+	$this->get( '/?s="exact match"' );
+	global $wp_query;
+
+	// Only 1 of the 3 posts should be found with hits for the words "exact match".
+	$this->assertEquals( 1, $wp_query->found_posts );
+} );
+
+it( 'tests that disabling phrase matching disables the feature.', function () {
```

Comment:
imo I wouldn't test the getter and the setter here. Instead, I'd add a test for the behavior when it's not enabled (it should match more things).

Reply:
for propriety, I am updating this comment with what we discussed in Slack. I discovered that ElasticPress (and thus VIP Search) uses a "phrase" clause OOtB in its "multi_match" queries. The nature of this phrase match is such that ElasticSearch will apply similar logic to it as what the phrase matching feature provides. In other words, VIP Search has "phrase matching" enabled by default. I will create a new issue shortly that would have this feature disabled if the feature is disabled via `disable_phrase_matching()`.

Reply:
in short, this test can't presently support testing the associated behavior, so I will add a comment and an issue and move on for now :)

Reply:
https://github.com/alleyinteractive/elasticsearch-extensions/issues/78